### DISCUSSION TITLE
how to align a picture to center with width < 100%

### DISCUSSION THREAD

**Discussion by luweizheng** [alshedivat/al-folio#1286] [Q&A]
hi,

I want to align an image to the center and the width is 80%.

```
{% include figure.html path="assets/blog/dataframe-analysis/dask-df.png" title="Dask DataFrame" class="img-fluid rounded z-depth-1" caption="Structure of a Dask dataframe" %}
```

There is a margin on the left. It appears like this:

![image](https://user-images.githubusercontent.com/57242864/227068150-2047d5a9-9c05-48a1-8c13-a881c4a16c4e.png)

One solution is to use `<div class="row">` and put some empty image placeholders on the left and the right. But it seems not so elegant.

So is there any quick method to align it to the center?

**Comment by george-gca** [comment]
Answered in #1514
